files/package.json

```diff
@@ -80,6 +81,9 @@
     "type": "addon",
     "main": "addon-main.cjs"
   },
+  "imports": {
+    "#src/*": "./src/*"
+  },
```

Comment:
I know we've talked about adding things like this for a while but it seems unrelated to this PR so can we remove it?

Reply:
moving to another PR